vterm/ops.go

```diff
@@ -160,6 +160,18 @@ func (v *VTerm) shiftCursorY(diff int) {
 	v.setCursorPos(v.Cursor.X, v.Cursor.Y+diff)
 }
 
+func (v *VTerm) setChar(x, y int, r rune, style ecma48.Style) {
+	if x >= v.w {
+		return
+	}
+	v.Screen[y][x] = ecma48.StyledChar{Rune: r, Style: style}
```

Comment:
I think we could reduce a bunch of mem allocs by using row/col-major order for the screen, but this is probably for a separate PR (one downside however, is that if there is a row with a bunch of empty space there will be some wasted memory).

Reply:
Yeah. I think we also need to look closer at the ecma48 spec because I'm unsure how text reflow is supposed to happen with manual cursor movement